staging/src/k8s.io/apiserver/pkg/admission/plugin/cel/interface.go

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cel
+
+import (
+	"time"
+
+	"github.com/google/cel-go/common/types/ref"
+
+	v1 "k8s.io/api/admission/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apiserver/pkg/admission/plugin/webhook/generic"
+)
+
+var _ ExpressionAccessor = &MatchCondition{}
+
+type ExpressionAccessor interface {
+	GetExpression() string
```

Comment:
```suggestion
type Expression interface {
    Expression() string
```
We don't put a `Get` in a getter function, in golang.

Reply:
Consider this a nit. it does not affect functionality.